src/_shared/pages/LandingPage/LandingPage.tsx

```diff
@@ -16,7 +16,7 @@ export const LandingPage = (): JSX.Element => {
         productLink=""
         menuLinks={[]}
       />
-      <StyledContainer>
+      <StyledContainer maxWidth="xl" disableGutters>
```

Comment:
Rather than having our own custom `maxWidth` CSS style set to 1260 px, let's use the nearest MUI default.